common/src/cookie/cookie.service.ts

```diff
@@ -55,6 +55,10 @@ export class CookieService<TKey extends string = string> {
     return this.parseCookieString(this.cookieString);
   }
 
+  public hasKey(key: TKey): boolean {

```

Comment:
it seems unused

Reply:
@astorozhevsky It will be used in real apps.